packages/shadydom/src/patch-native.js

```diff
@@ -94,8 +94,10 @@ const ParentNodeAccessors = [
 
 const ParentNodeMethods = [
   'querySelector',
-  'querySelectorAll'
-  // 'append', 'prepend'
+  'querySelectorAll',
+  'append',
+  'prepend',
+  'replaceChildren',
```

Comment:
Was this the right thing to do here? I'm not totally sure what this list is about. There seems to be some stuff later in this file that implies that they're on the wrong prototype in older Safari and that this is just about putting them in the correct spot - should I consider doing this for ChildNode in #390 too?

Reply:
I think the concern was [IE/Edge](https://github.com/webcomponents/polyfills/blob/4c539009f045723cee214f5212a5e42b2601f5f6/packages/shadydom/src/patch-native.js#L247).

It would be good to verify (perhaps just via one-off manual testing rather than in the code itself) that these properties do not exist *higher* on the prototype chain than where we patch them on any browser. This would cause the patch to fail.

Reply:
I checked the earliest version of the browsers implementing `append` (Edge 17, Firefox 49, Chrome 54, Safari 10) and they all have the descriptor on `Document`, `DocumentFragment`, and `Element` prototypes but not the `HTMLDocument`, `HTMLElement`, or `ShadowRoot` (if it exists) prototypes, so they should be alright assuming they haven't regressed and moved them around since.